src/adapters/leasing-adapter.ts

```diff
@@ -132,6 +132,23 @@ const getContactByContactCode = async (
   }
 }
 
+const getTenantByContactCode = async (
+  contactCode: string
+): Promise<AdapterResult<any, unknown>> => {
```

Comment:
Let's type the success to a Tenant type here? Made a similar comment in the leasing PR :)

Reply:
Will use after onecore-types/1560 is merged

Reply:
https://github.com/Bostads-AB-Mimer/onecore-core/pull/125/commits/c0c6d9245ba3a8636fd981711a14781cc4a0ae54